change/@ni-nimble-components-f58de85c-8a74-4a0b-b1c4-fdab36bfa94f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Could you add a note to [CSS Guidelines](https://github.com/ni/nimble/blob/main/packages/nimble-components/docs/css-guidelines.md) that describes our guidance to hide unused slots and how to target CSS to a slot?